netbox/dcim/models/device_components.py

```diff
@@ -1136,7 +1128,7 @@ class InventoryItem(MPTTModel, ComponentModel):
     clone_fields = ('device', 'parent', 'role', 'manufacturer', 'part_id',)
 
     class Meta:
-        ordering = ('device__id', 'parent__id', '_name')
```

Comment:
We can't arbitrarily drop ordering by parent Device, as it has ramifications unrelated to the tree hierarchy; for example, the order of results returned by the REST API endpoint.

Reply:
@jeremystretch can you provide the repo scenarios that this would effect?  CTE can only order on a single field (technical limitation) if we need to do two I'm not sure how to work around that.